### DISCUSSION TITLE
Is it possible to BULK add peers?

### DISCUSSION THREAD

**Discussion by mmomasters** [complexorganizations/wireguard-manager#78] [General]
I need to wireguard-manager.sh --add 100 clients

Like peer1 peer2 .... peer100

Is it possible?

Thank you!

**Comment by ghost** [comment]
Yes.
The client names have to be random for this to work, it will take way too long if the client name is entered manually.

**Comment by mmomasters** [comment]
I see and what if I write a bash script like:

wireguard-manager.sh --add peer1
wireguard-manager.sh --add peer2
........................................................................
wireguard-manager.sh --add peer100

Will this work?

**Comment by ghost** [comment]
```
echo -e "Client-Name" | ./wireguard-manager.sh --add
```
This will work and is already supported.
```
echo -e "\n" | ./wireguard-manager.sh --add
```
This will generate a random name for the client.
```
for i in {1..255}; do echo -e "\n" | ./wireguard-manager.sh --add; done
```
Loops 255 times and add random name on the client.
